### ISSUE TITLE
`--tajriba.store.file` flag doesn't work in production

### ISSUE THREAD

**Issue by JamesPHoughton** [empiricaly/empirica#383]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

If I run:
```bash
empirica --tajriba.store.file=$(pwd)/testTajriba.json
```
Then I can save my data in `testTajriba.json`.

But if I run:
```bash
empirica bundle
empirica serve my_experiment.tar.zst --tajriba.store.file=$(pwd)/testTajriba.json
```
then I get the following error message:
```bash
{"level":"fatal","error":"unknown flag: --tajriba.store.file","time":"2023-08-02T14:28:41-04:00","message":"empirica: failed to start"}
```

### Empirica Version

```text
Version: v1.4.6
SHA:     877d1e0
Branch:  main
Time:    2023-07-16T15:01:49Z

Client:  1.4.6
Server:  1.4.6
```

### What OS are you seeing the problem on?

Linux

### What browser are you seeing the problem on?

Does not apply

### Anything else?

Could be addressed with the same solution as https://github.com/empiricaly/empirica/issues/382

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by JamesPHoughton** [comment]
This is primarily useful in a situation where we have an external data volume that is going to be attached to the server to store persistent data, for example, I have an EFS volume mounted to /data on my server, and I want to save `tajriba.json` there.

Two reasons why this shouldn't (necessarily) be hardcoded in `empirica.toml`:

1. If I make changes to my code, and push a new version, the old tajriba will likely not work with the new version of the code. I should update the tajriba.json filename to something like `tajriba_build2.json` so that I won't have conflicts. We can do this by updating `empirica.toml` prior to bundling.
2. If I want to set up multiple servers running the same version of my code, and have them save to the same file volume, they will need to write to different tajriba.json files. If I could use the --tajriba.store.file flag in production, I could set the filename using an environment variable unique to the replication, and could use the same empirica bundle on both servers without needing to make any changes.

**Comment by npaton** [comment]
Fixed in 1.6.1. See [release notes](https://github.com/empiricaly/empirica/releases/tag/%40empirica%2Fcore%401.6.1) for details.
